docker/testdata/BUILD

```diff
@@ -344,13 +344,13 @@ load(
 )
 
 docker_bundle(
-   name = "bundle_to_push",
-   images = {
-     "us.gcr.io/convoy-adapter/bazel-test/{BUILD_USER}:foo": ":with_env",
-     "eu.gcr.io/convoy-adapter/bazel-test/{BUILD_USER}:bar": ":with_double_env",
-     "asia.gcr.io/convoy-adapter/bazel-test/{BUILD_USER}:baz": ":with_user",
-   },
-   stamp = True
+    name = "bundle_to_push",
+    images = {
+        "us.gcr.io/convoy-adapter/bazel-test/{BUILD_USER}:foo": ":with_env",
```

Comment:
It looks like this is both a functionality test, and implicitly a system test of the gcr regions (or whatever they're called).  Can you explicitly document that this is "a docker image from each GCR region" so that it's clear that we need to add a "moon.gcr.io" when we open the lunar base?